### ISSUE TITLE
Proposal for a new SC for hover (WCAG 3.0)

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag3#185]
### SC
When operating with a pointing device, a hover effect should be displayed on interactive elements that meets the following conditions:
* the click area of the interactive element should be visible,
* the cursor should be adequate and
* the contrast of the hover effect should be sufficient - either to the background or to the state without hover (at least 3:1).

### Purpose of hover visibility

* where the mouse cursor is located (hover effect is better visible than mouse cursor)
* interactive elements are recognizable as interactive
* type and operating options of the interactive elements are recognizable (via CSS styles for cursors: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor)
* status of interactive elements is recognizable (e.g. <code>cursor:not-allowed</code> for deactivated and read-only form fields)
* click area of the interactive elements is recognizable
* distinction between interactive elements is recognizable (e.g. in a tree: elements close to each other for fading in and out and for selecting treeitems)

### Benefits
* primarily cognitively impaired users
* visually impaired users (because the hover effect could be badly visible or interactive elements are badly recognizable as such, e.g. buttons have border with too little contrast, which is not a violation of 1.4.11, because buttons do not need a border)
* motorically impaired users using assistive technology that simulates pointer devices
* users who adjust the colors (High Contrast Mode, User Styles, color adjustment in the browser, etc.) because for them the interactive elements, their click area and their status are often difficult or impossible to recognize

### Level
AA or AAA

### Reason
The visibility of the hover is currently only partially covered by 1.4.11 (see https://github.com/w3c/wcag/issues/896). At 1.4.1 hover is only mentioned at https://www.w3.org/WAI/WCAG21/Techniques/general/G183. However, there is no criterion which requires a hover effect.
